utils/submit_tracks.py

```diff
@@ -272,29 +313,29 @@ def apply_template(genome_id: str, template_name: str, datafile: str = "") -> No
                 track_data["datafiles"][key] = f"{nameroot}-summary.bw"
```

Comment:
Why don't we add these data into yaml templates? These updates seem to fit in templates. 
In all apply_template()

Reply:
The datafile field is already present in yaml templates and in most cases is used in the track payload as-is. However, here we're updating the field dynamically to address the usecase where a track template is matches multiple datafiles (variable datafile names like `repeats.repeatmask*.bb`).